src/js/tech/html5.js

```diff
@@ -119,7 +119,10 @@ class Html5 extends Tech {
       this.on('loadstart', this[`removeOld${capitalType}Tracks_`]);
     });
 
-    if (this.featuresNativeTextTracks) {
+    //  edge only supports the `addtrack` event, so it
+    //  does not technically have native support, but we want to use the support
+    // that it does have.
+    if (this.featuresNativeTextTracks || browser.IS_EDGE) {
```

Comment:
The idea is that we proxy the native tracks always .I think what's missing is that in the `addtrack` handler we want to make sure that in the case of `this.featuresNativeTextTracks === false`, we only add the track for in-band tracks.